sdk/monitor/opentelemetry-exporter/src/export/exporter.ts

```diff
@@ -0,0 +1,150 @@
+import { Logger } from "@opentelemetry/api";
```

Comment:
I guess it makes sense that you'd use the logger from OT, though I'm not sure if we should have some story for if/how it would interact with `@azure/logger`